GameOfLife/View/DisplayGame.cs

```diff
@@ -26,5 +28,93 @@ public static void Display(int[,] args, Game game)
             Console.WriteLine($"Live cells: {game.CountOfLiveCells}");
             Console.WriteLine(GameMessages.pauseGame);
         }
+
+        /// <summary>
+        /// Makes view of multiple games field (2X4).
+        /// </summary>
+        /// <param name="games">List that holds all games.</param>
+        /// <param name="selectedEightGames">List that holds selected 8 games numbers.</param>
+        public static void DisplayMultipleGames(List<Game> games, List<int> selectedEightGames, int TotalAliveCells)
+        {
+            Console.Clear();
+            Console.WriteLine($"Total games: {games.Count}");
+            Console.WriteLine($"Iteration: {games[0].CountOfIteration}");
```

Comment:
Why you are using only first `CountOfIteration` ?

Reply:
As all games are running, it doesn't really matter which games' iteration is chosen in this case. It will be the same for all games.

Reply:
Seam a bit wasteful to add property for all games but use only one, but ok.